distutils/dist.py

```diff
@@ -1064,9 +1064,8 @@ def read_pkg_file(self, file):
 
         def _read_field(name):
             value = msg[name]
-            if value == 'UNKNOWN':
```

Comment:
I'm pretty sure this is here to replace UNKNOWN when found in an existing metadata file. How can we be confident this behavior isn't important for existing metadata?